assets/services.json

```diff
@@ -1599,6 +1599,15 @@
     },
+    {
+      "id": "soundcloud",
+      "name": "SoundCloud",
+      "rules": [
+          "||soundcloud.com^",
+          "||sndcdn.com^"
+      ],
+      "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\"  viewBox=\"0 0 24 24\" fill-rule=\"evenodd\" clip-rule=\"evenodd\"><path d=\"M19 17.75c2.07 0 3.75-1.68 3.75-3.75 0-2.07-1.68-3.75-3.75-3.75-.173 0-.344.012-.511.035-.73-2.337-2.913-4.035-5.489-4.035-.818 0-1.596.171-2.301.48-.273.119-.449.389-.449.687l0 9.583c0 .414.336.75.75.75l8 0zM7.25 8l0 9c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-9c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75zM4.25 10l0 7c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-7c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75zM1.25 12l0 5c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-5c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75z\"/></svg>"
```

Comment:
Hello @sandervankasteel
please also add fill=\"currentColor\" parameter

Reply:
@Sergey-Lyapin yes, I will add that :smile:

Reply:
Updated @Sergey-Lyapin.